charts/mediawiki/values.yaml

```diff
@@ -6,12 +6,13 @@ replicaCount:
 
 image:
   repository: ghcr.io/wbstack/mediawiki
-  tag: "1.37-7.4-20220513-fp-beta-0"
+  tag: "1.37-7.4-20220513-fp-beta-0" # TODO change this to new mw release!
   pullPolicy: IfNotPresent
 
 mw:
   settings:
     logToStdErr: false
+    allowedProxyCidr: ""
```

Comment:
looks good to me. I was wondering if we wanna add the comment about getting this value you mentioned in this comment https://phabricator.wikimedia.org/T309687#8011417 somewhere, either in this repo or wbaas-deploy? what do you think?

Reply:
Yeah sounds good to me!

Reply:
I just realized that we have README.md files for every chart in here, I put it there

Reply:
schweet